README.md

```diff
@@ -89,6 +118,22 @@ kurator install volcano
 kurator install prometheus --primary member1
 ```
 
+## Troubleshooting
```

Comment:
I would prefer this should not be in the Readme

Reply:
maybe move to wiki or under docs

Reply:
ok，move it to docs